Wikipedia/Code/BatchEditSelectView.swift

```diff
@@ -20,7 +24,7 @@ public class BatchEditSelectView: SizeThatFitsView {
     }
     
     fileprivate func updateMultiSelectIndicatorImage() {
-        let image = isSelected ? UIImage(named: "selected", in: Bundle.main, compatibleWith: nil) : UIImage(named: "unselected", in: Bundle.main, compatibleWith: nil)
+        let image = isSelected ? theme.multiSelectIndicatorImage : UIImage(named: "unselected", in: Bundle.main, compatibleWith: nil)
```

Comment:
any reason not to theme the image via tintColor = theme.colors.link like we do with the hint?

Reply:
Ya the white part inside the image.

Reply:
![simulator screen shot - iphone x - 2018-02-20 at 21 45 46](https://user-images.githubusercontent.com/9299317/36460675-70d46b9a-1687-11e8-9cde-f182fc383373.png)

works for me with
`let image = isSelected ? UIImage(named: "selected", in: Bundle.main, compatibleWith: nil) : UIImage(named: "unselected", in: Bundle.main, compatibleWith: nil)` in `updateMultiSelectIndicatorImage()` of `BatchEditSelectView`
and
`multiSelectIndicator?.tintColor = theme.colors.link` in `apply(theme: Theme)` of `BatchEditSelectView`

Reply:
@nambatee 
Hmm that pic doesn't appear to reflect the needed change (making the check image blue a lighter color). Did you switch the image to use "template" rendering in the assets catalog? My understanding of `template` style image handling w/imageview's tint is that the tint color replaces all non-clear parts of the image. In this case that makes even the white check become the tint color, which makes a solid circle. It was the first thing I tried.

Reply:
@nambatee The plus/add icon only has one non-transparent color so the tint trick works - ie the plus is just transparent (vs white), so using tint to change all the non-transparent parts does what we want.